eShopsolution.Data/Scripts/OrderDetail.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE OrderDetail (
+    Id INT PRIMARY KEY IDENTITY(1,1),  
+	TableName NVARCHAR(255) NOT NULL,
+	Time DateTime	NOT NULL,
+    Name NVARCHAR(255) NOT NULL,       
+    Quantity INT NOT NULL,            
+    Price FLOAT NOT NULL,              
+    Tongtien FLOAT NOT NULL           
+);
```

Comment:
thiết kế này chưa đáp ứng đủ yêu cầu, mình cần 1 table Order, rồi mới tới OrderDetail
CREATE TABLE Orders (
    OrderId INT PRIMARY KEY IDENTITY(1,1),
    TableId INT,
    DateOfOrder DATETIME,
    TotalAmount DECIMAL(10, 2)
);

CREATE TABLE OrderDetails (
    OrderDetailId INT PRIMARY KEY IDENTITY(1,1),
    OrderId INT,
    ProductId INT,
    Number INT,
    Price FLOAT NOT NULL,
    FOREIGN KEY (OrderId) REFERENCES Orders(OrderId)


Đây là ví dụ thôi, suy nghĩ rồi thiết kế lại DB chỗ này, giống bài luận văn hồi trước. Cái này cho làm ngày mai luôn á

Reply:
sẽ tạo ticket mới cho chỗ này